src/Shiny.Locations/Position.cs

```diff
@@ -37,21 +37,7 @@ public Distance GetDistanceTo(Position other)
 
 
         public override string ToString() => $"Latitude: {this.Latitude} - Longitude: {this.Longitude}";
-        //public bool Equals(Position other)
-        //{
-        //    if (other == null)
-        //        return false;
-
-        //    if (this.Latitude != other.Latitude)
-        //        return false;
-
-        //    if (this.Longitude != other.Longitude)
-        //        return false;
-
-        //    return false;
-        //}
-
-        public bool Equals(Position other) => (this.Latitude, this.Longitude).Equals((other?.Latitude, other?.Longitude));
+        public bool Equals(Position other) => other != null && (this.Latitude, this.Longitude).Equals((other.Latitude, other.Longitude));
```

Comment:
Comparison of `ValueTuple<double,double>` and `ValueTuple<double?,double?>` is always `false`. I didn't even notice this bug until I wrote the test.